backend/pkg/config/config.go

```diff
@@ -38,13 +38,16 @@ const (
 	EnableDBInfoLogs          = "ENABLE_DB_INFO_LOGS"
 	ViewRefreshIntervalEnvVar = "DB_VIEW_REFRESH_INTERVAL"
 
+	DefaultScanParallelism = "DEFAULT_SCAN_PARALLELISM"
```

Comment:
I think it will be useful to be able to configure it in the helm charts & values, WDYT?

Reply:
I think so too 👍🏻

Reply:
@FrimIdan I think so too 👍🏻 I pushed.